rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section5_envelope_tcd_master.json

```diff
@@ -14203,8 +14439,8 @@
         "Rule": 32,
         "Test": "c",
         "test_description": "Project has one building segment and includes a space with a roof surface. The roof surface solar reflectance in the proposed model is equal to 0.3, but the roof surface thermal emittance in the user model is equal to 0.7.",
-        "expected_rule_outcome": "pass",
-        "expected_raised_message_includes": "Roof surface solar reflectance is equal to the prescribed default value of 0.3 but differs from the solar reflectance in the user model",
+        "expected_rule_outcome": "undetermined",
+        "expected_raised_message_includes": "Fail if the thermal emittance in the user model is based on aged test data. roof surface solar reflectance is equal to the prescribed default value of 0.3 but differs from the solar reflectance in the user model",
```

Comment:
Can we add 0.7 at the end of the message? So, the message becomes ```'Fail if the thermal emittance in the user model is based on aged test data. roof thermal emittance is equal to the prescribed default value of 0.9  but differs from the thermal emittance in the user model 0.7.'```

Reply:
@yunjoonjung-PNNL thanks for your patience. I found that there were typos in both the rule and the ruletest. I corrected this.

Reply:
Thank you so much Juan!